### PR TITLE
Nrcan updates 20230123

### PR DESCRIPTION
**Pull request by ErikApption**

### CODE REVIEW

File: Portal/src/Datahub.Core/Utils/UrlPathSegment.cs [hunk 1]
```diff
@@ -3,4 +3,5 @@ namespace Datahub.Core.Utils;
 public static class UrlPathSegment
 {
     public const string PROJECTS = "projects";
+    public const string INITIATIVES = "initiatives";
```
Comment (yjmrobert) [inline Portal/src/Datahub.Core/Utils/UrlPathSegment.cs]: @ErikApption this has to be moved to config, can't be having it switch in the code

File: Portal/src/Datahub.Portal/Datahub.Portal.csproj [hunk 2]
```diff
@@ -74,6 +74,7 @@
     <PackageReference Include="DocumentFormat.OpenXml" Version="2.18.0" />
     <PackageReference Include="Localization.AspNetCore.TagHelpers" Version="0.6.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.21.0" />
```
Comment (yjmrobert) [inline Portal/src/Datahub.Portal/Datahub.Portal.csproj]: Should be able to infer the `Core` one

File: Portal/src/Datahub.Portal/Pages/Admin/InitiativeAdmin.razor [hunk 3]
```diff
@@ -22,7 +22,7 @@
                                     <MudStack Row>
                                         <MudText>@Localizer[$"{ADMIN_SECTION_LOCALIZATION_PREFIX}.InitiativeTrackerDesc"]</MudText>
                                         <MudSpacer/>
-                                        <MudButton Href=@($"/datahub/{UrlPathSegment.PROJECTS}") Variant=@Variant.Filled Color=@Color.Primary>
+                                        <MudButton Href=@($"/datahub/{UrlPathSegment.INITIATIVES}") Variant=@Variant.Filled Color=@Color.Primary>
```
Comment (yjmrobert) [inline Portal/src/Datahub.Portal/Pages/Admin/InitiativeAdmin.razor]: 😱

File: Portal/src/Datahub.Core/Utils/UrlPathSegment.cs (hunk 1 above)
Comment (yjmrobert) [reply Portal/src/Datahub.Core/Utils/UrlPathSegment.cs]: After many committees and fruitful deliberations, this is to be consolidated into the letter `w`

File: Portal/src/Datahub.Portal/Pages/Project/DataProjects.razor [hunk 4]
```diff
@@ -73,7 +73,7 @@
         await using var context = await _dbFactory.CreateDbContextAsync();
         _allProjects = context.Projects
             .Where(p =>
-                (p.Project_Status_Desc == Datahub_Project.ONGOING || p.Is_Featured) &&
+                (p.Project_Status == 1 || p.Is_Featured) &&
```
Comment (yjmrobert) [inline Portal/src/Datahub.Portal/Pages/Project/DataProjects.razor:76]: Need to move out of magic number territory

File: Portal/src/Datahub.Portal/i18n/localizations2.json [hunk 5]
```diff
@@ -173,7 +173,7 @@
   "Key_Activity_Additional_Information": "Key Activity Additional Information",
   "SFT With Attrition": "SFT With Attrition",
   "Data_Sensitivity": "Data Sensitivity",
-  "Project_Status_Values": "Project Status",
+  "Project_Status_Values": "Initiative Status",
```
Comment (yjmrobert) [inline Portal/src/Datahub.Portal/i18n/localizations2.json]: Needs to go in `localizations2.nrcan.json`

File: infra/azure-pipelines.yml [hunk 6]
```diff
@@ -82,7 +82,7 @@ steps:
     azureSubscription: 'datahub-dev-service-connection'
     appType: 'functionApp'
     appName: 'dh-portal-func-dev'
-    package: '$(System.DefaultWorkingDirectory)/Datahub.Graph.Functions/**/*.zip'
+    package: '$(System.DefaultWorkingDirectory)/Portal/src/Datahub.Functions/**/*.zip'
 
 - task: AzureAppServiceSettings@1
   displayName: Azure App Service Settings
```
Comment (yjmrobert) [inline infra/azure-pipelines.yml:88]: @simon-wang-gc , did you want to merge this into SSC's Datahub project?

**Review by yjmrobert — CHANGES_REQUESTED** [review]

File: Portal/src/Datahub.Portal/Pages/Admin/InitiativeAdmin.razor (hunk 3 above)
Comment (ErikApption) [reply Portal/src/Datahub.Portal/Pages/Admin/InitiativeAdmin.razor]: It's all now a boring /w/

File: Portal/src/Datahub.Portal/i18n/localizations2.json (hunk 5 above)
Comment (ErikApption) [reply Portal/src/Datahub.Portal/i18n/localizations2.json]: split into ssc/nrcan files

File: Portal/src/Datahub.Portal/Datahub.Portal.csproj (hunk 2 above)
Comment (ErikApption) [reply Portal/src/Datahub.Portal/Datahub.Portal.csproj]: done

File: Portal/src/Datahub.Portal/Pages/Project/DataProjects.razor (hunk 4 above)
Comment (ErikApption) [reply Portal/src/Datahub.Portal/Pages/Project/DataProjects.razor:76]: created bug for Nabeel

File: Portal/src/Datahub.Portal/i18n/localizations2.json (hunk 5 above)
Comment (ErikApption) [reply Portal/src/Datahub.Portal/i18n/localizations2.json]: done

File: infra/azure-pipelines.yml (hunk 6 above)
Comment (yjmrobert) [reply infra/azure-pipelines.yml:88]: Not applicable to SSC's infra

**Pull request merged**